crates/cast/src/starknet_commands/declare.rs

```diff
@@ -86,11 +91,17 @@ pub async fn declare(
     let casm_class_hash = casm_contract_definition.class_hash()?;
 
     let declaration = account.declare(Arc::new(contract_definition.flatten()?), casm_class_hash);
-    let execution = if let Some(max_fee) = max_fee {
+    // todo: refactor setting max_fee and nonce
```

Comment:
Let's add an issue for this

Reply:
https://github.com/foundry-rs/starknet-foundry/issues/1303